### ISSUE TITLE
Pandas AttributeError when rendering Molecule in DataFrame

### ISSUE THREAD

**Issue by enricogandini** [rdkit/rdkit#3701]
### Description

This issue occurs when, in a Jupyter Notebook (in JupyterLab), I display a DataFrame,
after activation of Molecule Object rendering.

If I use `PandasTools.AddMoleculeColumnToFrame` to add a Molecule column to an existing DataFrame, or if I use `Chem.MolFromSmiles` and then `PandasTools.RenderImagesInAllDataFrames(images=True)`, whenever I display a DataFrame I get:
```python
AttributeError: module 'pandas.io.formats.format' has no attribute '_get_adjustment'
```
The DataFrame is not displayed correctly in the notebook; it is just printed instead.

Single molecules are still rendered correctly; for instance, using
```python
df.at[0, "molecule"]
```
I get a correctly rendered picture of the Molecule object at index 0 of the DataFrame.

### Expected Behaviour
I would expect a correctly displayed DataFrame in the notebook. The column containing the Molecule Object should be rendered as images of each molecule.

### Configuration
* RDKit version 2020.09.3
* Pandas version 1.2.0
* Python version 3.7.9
* JupyterLab version 3.0.1
* OS: Ubuntu 18.04
* RDKit installed with conda, from channel conda-forge.

### Additional info
Please see the attached notebooks (in PDF format).
[bug_display_molecules_1.pdf](https://github.com/rdkit/rdkit/files/5781553/bug_display_molecules_1.pdf)
[bug_display_molecules_2.pdf](https://github.com/rdkit/rdkit/files/5781554/bug_display_molecules_2.pdf)

**Comment by greglandrum** [comment]
Thanks for the bug report @enricogandini
A request: it would make verifying/reproducing your bug reports easier if you could provide the ipynb files directly. I know that github doesn't make it easy to upload these, but you can either add `.txt` to the end of the filename: `foo.ipynb.txt` or gist them and then provide a link.

FWIW: this does work with pandas v1.1.2: https://gist.github.com/greglandrum/3b9da87a20e9b0d4827bbf1e64f68056

I'll create an environment with pandas 1.2 and see if I can track it down.

**Comment by enricogandini** [comment]
If still needed, here are the original Notebooks (with `.ipynb.txt` extension):
[bug_display_molecules_1.ipynb.txt](https://github.com/rdkit/rdkit/files/5801239/bug_display_molecules_1.ipynb.txt)
[bug_display_molecules_2.ipynb.txt](https://github.com/rdkit/rdkit/files/5801240/bug_display_molecules_2.ipynb.txt)

I am sorry, I am still not very familiar with GitHub. I solved this issue using the GitHub interface, so I think I did it in a personal copy of the RDKit repository, and I am not sure if you can see it.
Basically, in file `PandasTools.py`, I changed all occurrences of `pandas.io.formats.format._get_adjustment` to `pandas.io.formats.format.get_adjustment`; in Pandas 1.2, they removed the underscore prefix of the `get_adjustment` method, but the change is not documented in the release notes.
I can confirm that everything worked perfectly with older versions of Pandas.
